crates/oxc_linter/src/rules/unicorn/prefer_set_size.rs

```diff
@@ -157,5 +168,11 @@ fn test() {
         r"[...new /* comment */ Set(array)].length",
     ];
 
-    Tester::new(PreferSetSize::NAME, pass, fail).test_and_snapshot();
+    let fix = vec![
+        (r"[...new Set(array)].length", r"new Set(array).size"),
+        (r"[...new Set(array),].length", r"new Set(array).size"),
+        (r"[...(( new Set(array) ))].length", r"(( new Set(array) )).size"),
```

Comment:
can you add tests that use parenthesis, semicolons, type casting, etc?

Reply:
done, let me know if there're any other cases you want me to add